paddle/fluid/operators/elementwise/elementwise_mul_op.h

```diff
@@ -17,10 +17,89 @@ limitations under the License. */
 #include "paddle/fluid/operators/elementwise/elementwise_op_function.cu.h"
 #include "paddle/fluid/operators/elementwise/elementwise_op_function.h"
 #include "paddle/fluid/operators/math/blas.h"
+#include "paddle/fluid/platform/cpu_info.h"
 
 namespace paddle {
 namespace operators {
 
+class ElementwiseMulOp : public framework::OperatorWithKernel {
+ public:
+  using framework::OperatorWithKernel::OperatorWithKernel;
+  using Tensor = framework::Tensor;
+
+  void InferShape(framework::InferShapeContext* ctx) const override {
+    PADDLE_ENFORCE(ctx->HasInput("X"),
```

Comment:
PADDLE_ENFORCE is not recommended. Please use PADDLE_ENFORCE_EQ/NE/GT/GE/LT/LE or PADDLE_ENFORCE_NOT_NULL or PADDLE_ENFORCE_CUDA_SUCCESS instead.
http://ci.paddlepaddle.org/viewLog.html?buildId=214325&buildTypeId=YYTest_PrCiCpuPy2&tab=buildLog&branch_YYTest=__myBranchesProvider-myBranches__

Reply:
You're right, sorry I have cherry-picked the code from the old PR #17104 which used old PADDLE_ENFORCE system and forgotten to replace it. On my way to do it now.